python/GafferUI/EditMenu.py

```diff
@@ -94,18 +94,12 @@ def appendDefinitions( menuDefinition, prefix="" ) :
 # filtered within the GraphEditor.
 def scope( menu ) :
 
+	graphEditor = None
```

Comment:
Move to line 100, so all initialisation for `graphEditor` is in one place.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3372/commits/5129e4597c2dc9c844e6e05e387032c3e65588fc